.github/workflows/test.yaml

```diff
@@ -21,3 +21,5 @@ jobs:
         run:  .github/scripts/test.sh
       - name: Format Check
         run:  .github/scripts/format-check.sh
+      - name: Docs Link Check
+        run: .github/scripts/link-check.sh
```

Comment:
Run the link checker as part of the `test` workflow (run for all commits to `develop`).